src/components/common/footer/footer.scss

```diff
@@ -1,3 +1,22 @@
+.links-container {
+  display: flex;
+  justify-content: center;
+
+  a {
+    padding: 5px;
+    color: #7A6DAC;
+
+    
+  }
+  a:hover {
+    color: #050505;
+    transform: scale(.98)
+  }
```

Comment:
hover looks good

Reply:
I think you can get away with a bigger difference in scale if you want. Up to you.

Reply:
Yeah I didn’t change anything in the footer. If I go back in I can take out that doc surrounding {CONTENT} though. 

Yeah I was just going to see what you thought and do the same scale on the buttons as well if you think that would be a good idea

Reply:
If it's not part of your changes, ignore it. We can create a new issue if we want down the line but it's not so important.

Reply:
Okay I think I might have deleted the div. But does everything with the links good on your end?